Assets/UI/CQUICommon.lua

```diff
@@ -17,35 +17,33 @@
 g_bIsRiseAndFall    = Modding and Modding.IsModActive("1B28771A-C749-434B-9053-D1380C553DE9"); -- Rise & Fall
 g_bIsGatheringStorm = Modding and Modding.IsModActive("4873eb62-8ccc-4574-b784-dda455e74e68"); -- Gathering Storm
 
-
 -- ===========================================================================
 -- Debug support
 -- ===========================================================================
-
 CQUI_ShowDebugPrint = false;
 
 function print_debug(...)
     if CQUI_ShowDebugPrint then
-        print(...);
+        local str = "[CQUI] " .. table.concat({...}, " ");
```

Comment:
This is a potential source of errors.
If you use multi-args then table.concat will not work if one of them is not string.
So, whoever wants to use print_debug should know that it takes only strings, which is bad assumption.
I think this will suffice: print("[CQUI]", ...)